model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -117,6 +117,25 @@ class LicenseInfoResolver(
                         )
                     )
                 }
+
+                licenseInfo.declaredLicenseInfo.copyrightHolders.takeIf { it.isNotEmpty() }?.let {
+                    locations.add(
+                        ResolvedLicenseLocation(
+                            provenance = UnknownProvenance,
+                            location = UNDEFINED_TEXT_LOCATION,
+                            appliedCuration = null,
+                            matchingPathExcludes = emptyList(),
+                            copyrights = licenseInfo.declaredLicenseInfo
+                                .copyrightHolders.mapTo(mutableSetOf()) { copyrightHolder ->
```

Comment:
Take benefit from using `let` / `also` above, and replace `licenseInfo.declaredLicenseInfo.copyrightHolders` with `it` here.

Reply:
done